core/src/mindustry/world/blocks/defense/turrets/BaseTurret.java

```diff
@@ -51,6 +51,9 @@ public void init(){
             coolant.update = false;
             coolant.booster = true;
             coolant.optional = true;
+
+            //json parsing does not add to consumes
+            if(!hasConsumer(coolant)) consume(coolant);
```

Comment:
What other blocks (if any) have special singular consumes that need to be added in case of json parsing?